utilities/pytest_utils.py

```
@@ -46,6 +46,7 @@
 )
 
 LOGGER = logging.getLogger(__name__)
+test_edit
```

Ruff (0.15.17)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)